### DISCUSSION TITLE
Same navigation for page and docs

### DISCUSSION THREAD

**Discussion by fsmeier** [mearashadowfax/ScrewFast#107] [Ideas]
Heyhey, 

what would be nice is to have the same look&feel for the docs part as for the main part. Mainly about the navigation. This seems to be a rather complex topic. Also with 2 separate systems used for the dark-bright-theme switcher :D 

But maybe this is something some more could find useful. 

All the best,
Florian

**Comment by mearashadowfax** [comment]
Hi @fsmeier! Thank you for your feedback! Your idea to unify the look and feel between the documentation and the main site is indeed a great one. I've updated the theme component to match the main website. Integrating the full navbar is more complex, but we might tackle it later.
